### PR TITLE
feat(gdpr-api): add translations to gdpr data

### PR DESCRIPTION
**Pull request by nicobav**
Refs LINK-2012

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr913.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr913.api.dev.hel.ninja** :rocket::rocket::rocket:

File: helevents/tests/test_gdpr_api_get.py [hunk 1]
```diff
@@ -91,6 +103,16 @@ def _get_signup_group_profile_data(signup_group: Optional[SignUpGroup]) -> dict:
     return profile_data
 
 
+def _get_field_translations(obj: object, field_name: str) -> list:
+    return [
+        {
+            "key": f"{field_name}_{lang}".upper(),
+            "value": getattr(obj, f"{field_name}_{lang}"),
```
Comment (charn) [inline helevents/tests/test_gdpr_api_get.py:110]: Maybe we should skip translation which don't have a value for a specific language? Just thinking that like this well probably end up most translations being `None`.

Comment (nicobav) [reply helevents/tests/test_gdpr_api_get.py:110]: yes, your comment applies to the actual logic also. Yup perhaps the fields not having translation could just be left out 🤔

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr913.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Pull request merged**